test/org/sagebionetworks/bridge/models/studies/ConsentSignatureTest.java

```diff
@@ -207,6 +308,41 @@ public void jsonHappyCaseWithImage() throws Exception {
     }
     
     @Test
+    public void signatureContainsSignedOnValue() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+        JsonNode jsonNode = JSON_OBJECT_MAPPER.readTree(json);
+
+        ConsentSignature sig = ConsentSignature.createFromJson(jsonNode, UNIX_TIMESTAMP);
+        assertEquals(UNIX_TIMESTAMP, sig.getSignedOn());
+    }
+    
+    @Test
+    public void existingSignatureJsonDeserializesWithoutSignedOn() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(json, ConsentSignature.class);
+        assertEquals("test name", sig.getName());
+        assertEquals("1970-01-01", sig.getBirthdate());
+        assertEquals(0, sig.getSignedOn());
```

Comment:
Is this possible? Wouldn't the validator fail because of 0 signedOn?

Also, validator test cases for 0 and negative signedOn?